spec/inputs/heartbeat_spec.rb

```diff
@@ -1 +1,92 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/heartbeat"
+
+def fetch_event(config, number)
+  pipeline = LogStash::Pipeline.new(config)
+  queue = Queue.new
+  pipeline.instance_eval do
+    @output_func = lambda { |event| queue << event }
+  end
+  pipeline_thread = Thread.new { pipeline.run }
+  counter = 0
+  begin
+    event = queue.pop
+    pipeline_thread.join
+    counter += 1
+  end until counter == number
+  return event
+end
```

Comment:
Since the `fetch_event` is more like a support method than an actual test I suggest we move this method to a helper file `spec/support/helper.rb`, we could refactor the helper into `devutils` with a simple search on the support files? thoughts?